sources/packages/backend/apps/db-migrations/src/sql/ApplicationOfferingChangeRequests/Add-col-student-consent.sql

```diff
@@ -0,0 +1,7 @@
+-- Create student consent column. 
+ALTER TABLE
+  sims.application_offering_change_requests
+ADD
+  COLUMN student_consent BOOLEAN DEFAULT false;
```

Comment:
why should there be a default? I believe that can be misinterpreted and by default null should be fine, IMO

Reply:
That's right. null should be fine as default.